### PR TITLE
drop script loading from GOPATH

### PR DESCRIPTION
**Pull request by MatousJobanek**
Two fixes:
1. drops script loading from GOPATH
2. load properly the local files (based on the script directory)

### CODE REVIEW

File: scripts/ci/manage-host-operator.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ read_arguments() {
 
 set -e
 
-read_arguments $@
+#read_arguments $@
```
Comment (xcoulon) [inline scripts/ci/manage-host-operator.sh:88]: should this be removed?

**Review by xcoulon — APPROVED** [review]

File: scripts/ci/manage-host-operator.sh (hunk 1 above)
Comment (MatousJobanek) [reply scripts/ci/manage-host-operator.sh]: whoopsee, it's a leftover from my local testing :hand_over_mouth: 
good catch, thanks a3e7386

Comment (MatousJobanek) [reply scripts/ci/manage-host-operator.sh]: https://issues.redhat.com/browse/KUBESAW-200

**Review by mfrancisc — APPROVED** [review]
Thanks for the fix 👍

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**